index.html

```diff
@@ -0,0 +1,178 @@
+  </head>
+  <body>
+    <header>
+        <div id="modal__container" class="modal__container">
+            <div class="modal__header">
+              <div></div>
+              <i id="btn__close" class="fa fa-times modal__icon" aria-hidden="true"></i>
+            </div>
+            <div class="modal__content">
+              <a href="About.html"  id="modal__titles1"> About</a>
+              <a href="about.html"  id="modal__titles2"> Program</a>
+              <a href="#contact" id="modal__titles3"> Join</a>
+              <a href="#contact" id="modal__titles3"> Sponsor</a>
+              <a href="#contact" id="modal__titles3"> News</a>
+            </div>
+        </div>
+        <i id="btn__open" class="fa fa-bars" aria-hidden="true"></i>
+      <nav class="socail_nav hide">
+        <div class="socail_nav_div">
+          <a href="#"
+            ><img src="./images/facebook.png" alt="facebook_icon"
+          /></a>
+          <a href="#"><img src="./images/twitter.png" alt="twitter_icon" /></a>
+          <span> English</span>
+          <span>MyPage</span>
+          <span>Logout</span>
+        </div>
+      </nav>
+      <nav class="navbar hide">
+        <div class="logo">
+          <a href="index.html">
+            <img src="./images/logo.png" alt="logo" srcset=""
+          /></a>
+        </div>
+        <ul class="nav_menu_list">
+          <li class="nav_menu_item"><a id="about" href="About.html">About</a></li>
+          <li class="nav_menu_item"><a href="#">Program</a></li>
+          <li class="nav_menu_item"><a href="#">Join</a></li>
+          <li class="nav_menu_item"><a href="#">Sponsor</a></li>
+          <li class="nav_menu_item"><a href="#">News</a></li>
+          <li class="nav_menu_item"><a id="ticket" href="#">Tickets</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="intro">
+      <h2 class="intro_subtitle">"Hello! Gaming world"</h2>
+      <h1 class="intro_title">
+        CREATIVE COMMONS <br />GLOBAL SUMMIT<br />
+        2023
+      </h1>
+      <p class="intro_bio">
+        A joyful celebraion believing in the value of opennes and sharing
+        creating a positive change with people from over 80 countries is takinng
+        place in October, in Korea.
+      </p>
+      <p class="event_time" id="event-time">2023.10.15(THU) ~ 16(FRI)</p>
+      <p class="location">
+        @ National Mueseum of Korea, Art Center Nabi and more
+      </p>
+    </section>
+    <section class="main">
+      <h2 class="main_header">Main Program</h2>
+      <span class="main_header_underline"></span>
+      <ul class="main_list">
+        <li class="main_list_card">
+          <img
+            class="main_list_img"
+            src="./images/info_icon1.png"
+            alt="info1_pic"
+            srcset=""
+          />
+          <h3 class="main_list_header">Lecture</h3>
+          <span class="main_list_bio">
+            Lorem ipsum dolor sit, amet adipisicing elit.consectetur adipisicing
```

Comment:
- [ ] please kindly add the sizes(height) of the cards in the mobile version so they can be more visible and follow the original design as well.
![Screenshot (999)_LI](https://user-images.githubusercontent.com/65298048/234508029-801531e9-2b64-4b74-ab1a-34dd5fa54187.jpg)
original
![Screenshot (1000)](https://user-images.githubusercontent.com/65298048/234508169-3717d579-2ac0-4515-a54d-f75436cb1cff.png)